src/js/flows/__snapshots__/submitSearch.test.js.snap

```diff
@@ -15,6 +15,13 @@ Array [
   "VIEWER_END_STATUS",
   "HANDLERS_ABORT",
   "HANDLERS_REGISTER",
+  "CHART_STATUS",
+  "VIEWER_STATUS",
+  "CHART_STATUS",
+  "HANDLERS_REMOVE",
+  "VIEWER_STATUS",
+  "VIEWER_END_STATUS",
+  "HANDLERS_REMOVE",
```

Comment:
The submitSearch test now uses a "Mock Zealot" that more successfully mimics reality. It responds with more real payloads, hence this addition in the snapshots.